csrc/quantization/fused_kernels/layernorm_utils.cuh

```diff
@@ -0,0 +1,302 @@
+#pragma once
+
+/**
+ * __device__ layernorm utilities.
+ */
+
+#include "vectorization.cuh"
+#include "quant_conversions.cuh"
+
+namespace vllm {
+
+// has_residual must be true, if residual is not a nullptr
+template <typename scalar_t, bool has_residual = false>
+__device__ void compute_rms(float* rms, scalar_t const* __restrict__ input,
+                            int const hidden_size, float const epsilon,
+                            scalar_t const* __restrict__ residual = nullptr) {
+  int const token_offset = blockIdx.x * hidden_size;
```

Comment:
this should be an int64_t

Reply:
Thanks for catching this. Updated the `int`s to `int32_t` / `int64_t` 👍